apps/api/package.json

```diff
@@ -99,8 +99,7 @@
     "twilio": "^4.14.1",
     "uuid": "^8.3.2",
     "zod": "^3.23.8",
-    "zod-to-json-schema": "^3.23.3",
-    "json-schema-to-ts": "^3.0.0"
```

Comment:
This dependency (`json-schema-to-ts`) got removed, just confirming it's intentional?

Reply:
Double entry: https://github.com/novuhq/novu/pull/7073/files#diff-2c40985d6d91eed8ae85ec1c8e754a85984ee32e156a600d2b7a467423d7e338L77